validate.js

```diff
@@ -0,0 +1,63 @@
+const bfe = require('ssb-bfe')
+
+/**
+ * Validate a single meta feed message.
+ *
+ * @param {Array | string} contentSection - an array of `content` and `contentSignature` or an encrypted string
+ * @returns {Object | true} an `Error` object or `true` in the case of successful validation
+ */
+exports.validateSingle = function (contentSection) {
+  if (contentSection === null || contentSection === undefined)
+    return new Error(
+      `invalid message: contentSection cannot be null or undefined`
+    )
+
+  // check if content is (maybe) encrypted
+  if (typeof contentSection === 'string')
+    return new Error(
```

Comment:
I'm not sure an `Error` return type makes the most sense here but I also don't want to make the return handling unnecessarily complex.

My thinking is that the caller checks the return type: if `true`, all good; if `Error`, check the `err.message` and act accordingly (could be used to catch encrypted input case).